src/RatingAPILayer.jsx

```diff
@@ -91,6 +92,7 @@ class RatingAPILayer extends React.PureComponent {
         placeholderSymbol={placeholderSymbol}
         onClick={this.handleClick}
         onHover={this.handleHover}
+        {...other}
```

Comment:
Please remove passing other props. Let's open an issue to discuss any specific additions you'd like to see to the API. I don't really like the idea of mass passing of undocumented props.